Sample code for Icon class

Piinks opened the pull request:
Added sample code for a simple Icon, ref: #21136

Piinks committed:
Added sample code for a simple Icon, ref: #21136

Piinks committed:
Removed duplicate line.

Piinks committed:
Fixed incoreect parameter.

goderbauer commented on packages/flutter/lib/src/widgets/icon.dart:
````diff
@@ -21,6 +21,20 @@ import 'icon_theme_data.dart';
+/// ```dart
+/// Icon(
+///   Icons.add,
+///   color: Colors.white,
````
White may not be the best choice. Most apps probably have a white background (it's the default after `flutter create`) and if somebody copy & pastes this example they will not see the icon. Maybe chose a color that's less likely to be the background?

goderbauer commented on packages/flutter/lib/src/widgets/icon.dart:
```diff
@@ -21,6 +21,20 @@ import 'icon_theme_data.dart';
 ///
+/// {@tool sample}
+///
+/// This example shows how to use [Icon] to create a simple icon in your flutter
```
Also: please remove "in your flutter code", it's redundant. See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

However, maybe you can describe with a word or two what kind of icon the user can expect from this example code.

Piinks committed:
Update icon.dart

Removed empty prose and added better context for sample code. Changed sample code for better visibility in execution of default app themes.

goderbauer reviewed (approved):
LGTM

Piinks committed:
Alas, Trailing space.

Nixed trailing space in comments.
